src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -38,7 +38,11 @@ export class AuthenticationApiImpl implements AuthenticationApi {
   private readonly authServerUrl: string;
   private readonly authProcessId: string;
 
-  async startAuthentication(healthcareProfessionalId: string | undefined, firstName: string, lastName: string, email: string, recaptcha: string, mobilePhone?: string): Promise<AuthenticationProcess | null> {
+  async startAuthentication(healthcareProfessionalId: string | undefined, firstName: string, lastName: string, recaptcha: string, email?: string, mobilePhone?: string): Promise<AuthenticationProcess | null> {
+    if (email == undefined && mobilePhone == undefined) {
```

Comment:
Could `!email && !mobilePhone` be used as the condition to prevent empty strings from being given as input?

Reply:
Yes, we can : Just edited